warehouse/models/gtfs_views/_gtfs_views.yml

```diff
@@ -841,6 +841,49 @@ models:
         description: Field in validation output from GTFS validator
         tests:
           - not_null
+  - name: reports_daily_service_route_stops
+    description: |
+      New table to consolidate and improve metric generation for reports.
```

Comment:
I would refactor this a bit to be a generic description of the table content / purpose -- this description will persist in general, so table will no longer be "new".